### PR TITLE
Update & streamline SLAM slides

### PR DESCRIPTION
**Pull request by ipa-kut**

### CODE REVIEW

File: slides/Navigation/source/SLAM.html [hunk 1]
```diff
@@ -38,177 +38,140 @@
...
-  * Graph-based SLAM, e.g. Pose graph optimization (Cartographer, SLAM toolbox, Karto SLAM)
+* Kalman Filters - [source](http://ais.informatik.uni-freiburg.de/teaching/ws12/mapping/pdf/slam08-kf-wrapup.pdf)
+    - Probabilistic, Bayes filter, best for Linear Models and Gaussian distributions
+    - Further specialisations - Extended KF, Unscented KF, Extended Information Filter (EIF), Sparse efficiency
+* Particle Filters - [source](http://ais.informatik.uni-freiburg.de/teaching/ws12/mapping/pdf/slam09-particle-filter.pdf)
+    - Probabilistic, Recursive Bayes filter, also appropriate for non-Gaussian distributions
+    - AKA Monte Carlo Localisation - most commonly used
+* Graph based - [source](http://ais.informatik.uni-freiburg.de/teaching/ws12/mapping/pdf/slam17-ls-landmarks.pdf)
+    - Deterministic, Graph of a problem with nodes for poses & edges for spatial constraints between poses
+    - Build the graph and find a node configuration that minimizes the error introduced by the constraints
```
Comment (ipa-rwu) [inline slides/Navigation/source/SLAM.html:62]: Shall we mention some packages here?

File: slides/Navigation/source/SLAM.html [hunk 2]
```diff
@@ -38,177 +38,140 @@
...
-
-<img src="../static/voxel_resolutions.png" alt="point_cloud_tree" style="height:150px" class="image-left"/>
-<img src="../static/octree.png" alt="point_cloud_tree" style="height:150px" class="image-right"/>
-
-Occupied voxels are displayed in resolutions 0.08 m, 0.64 , and 1.28 m.
+Fig 1: Closed environment with Robot (Yellow) and Obstacle (Blue)   
+Fig 2: When robot moves from A to B and is perfectly tracked (Perfect sensor & odometry)   
+Fig 3: When robot moves from A to B but has pose inaccuracy (Imperfect sensor &/or odom)   
+Fig 4: Mapping with pose inaccuracy (Imperfect sensor & odom)   
+Fig 5: Loop closure problem:  After one loop, Starting Pose A = Ending Pose B in real world, but not in map.
```
Comment (ipa-rwu) [inline slides/Navigation/source/SLAM.html:169]: Nice! much clear!

File: slides/Navigation/source/SLAM.html (hunk 1 above)
Comment (ipa-kut) [reply slides/Navigation/source/SLAM.html:62]: Its a good idea, but we would need to dig into each package to see what type of algorithm they use, and I just ran out of time to do it. If you are able to do so, please go ahead!

Comment (ipa-rwu) [reply slides/Navigation/source/SLAM.html:62]: I will merge first. We can do later

**Pull request merged**